packages/contentstack-seed/package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "@contentstack/cli-cm-seed",
   "description": "create a Stack from existing content types, entries, assets, etc.",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "author": "Contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
-    "@oclif/core": "^1.21.0",
```

Comment:
@aman19K We need @oclif/core in the seed command. Because we using it in the packages/contentstack-seed/src/commands/cm/stacks/seed.ts file.

Reply:
@antonyagustine done